src/plots/vote_plot.py

```diff
@@ -74,3 +81,8 @@ def split_by_ratings(data: pd.DataFrame) -> list[list]:
 
 fig.tight_layout()
 plt.show()
+if EXPORT_PLOTS:
+    plt.savefig(
+        "../../report/img/rating_vote_heatmap_excluding_no_votes.svg",
```

Comment:
There seems to be something wrong with the heatmap svg generated. It won't render here, in my IDE previewer or using an online SVG previewer

Reply:
FIXED! 🔧